src/Parser/ShadowPriest/Modules/Spells/VoidformsTab.js

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChartistGraph from 'react-chartist';
+import Chartist from 'chartist';
+
+import 'chartist-plugin-legend';
+
+import './VoidformsTab.css';
+
+const formatDuration = (duration) => {
+  const seconds = Math.floor(duration % 60);
+  return `${Math.floor(duration / 60)}:${seconds < 10 ? `0${seconds}` : seconds}`;
+};
+
+
+const VoidformsTab = ({voidforms=[], ...modules}) => {
+  if(voidforms.length === 0) return null;
+  return (<div className="voidforms">{voidforms.map((voidform, i) => <VoidformGraph key={i} {...voidform} {...modules} />)}</div>);
+};
+
+VoidformsTab.propTypes = {
+  voidforms: PropTypes.array.isRequired,
+  fightEnd: PropTypes.number.isRequired,
+};
+
+const MAX_SECONDS = 65;
+
+const VoidformGraph = ({start, ended, stacks, lingeringInsanityStacks, fightEnd, mindbenders, voidTorrents, dispersions}) => {
```

Comment:
I would prefer if you put this component into its own file, even if that means the VoidformsTab file would be very small right now.